configs/letsencrypt/create-self-signed.sh

```diff
@@ -25,8 +25,8 @@ if [ "${#domains[@]}" -lt "1" ]; then
   exit 1
 fi
 
-if ! [ -x "$(command -v docker-compose)" ]; then
-  echo 'Error: docker-compose is not installed.' >&2
+if ! [ -x "$(command -v docker compose)" ]; then
```

Comment:
I don't think this works tbh. `docker` is the command / binary. `compose` is just an argument. The more robust way I think is to check the exit code like I did in `setup.sh`